src/runtime/server/settings.ts

```diff
@@ -9,6 +10,60 @@ export type PlaygroundSettings = {
   settings?: Omit<Partial<Exclude<PlaygroundRenderPageOptions['settings'], undefined>>, 'general.betaUpdates'>
 }
 
+export type CorsSettings = {
+  /**
+   * Configures the Access-Control-Allow-Origin CORS header. Possible values:
+   *
+   * Boolean - set origin to true to reflect the request origin, as defined by req.header('Origin'), or set it to false to disable CORS.
```

Comment:
`false` would be shorthand for `{ enabled: false }`

Reply:
> Not sure what "reflect the request origin" means.

It means that whatever value was set in the `Origin` header request will be "reflected" in the `Access-Control-Allow-Origin` response header.

eg:

Request: `Origin: www.example.com`
Response: `Access-Control-Allow-Origin: www.example.com`

Reply:
> I think I suggest a new field enabled to control enabling/disabling. Depending on clarification to above question.

Making `origin: false` is not specifically designed to disable the feature, it's just a side-effect, where disabling it will make express no longer respond with an `Access-Control-Allow-Origin` header, thus disabling CORS.

You can already disable CORS by settings `cors: false`, I'm not sure it's worth adding yet another `enabled` property ?

Reply:
Added `enabled` field

Reply:
> IIUC it's more or less equivalent to a wildcard.

Yeah seems like it... what's the point 🤔 haha